scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
```

Comment:
In Ruby, we don’t need to specify the brackets when defining methods without arguments. So by convention, we usually don’t include them.